tests/specs/query.js

```diff
@@ -61,26 +61,12 @@
             };
         });
 
-        afterEach(function (done) {
+        afterEach(function () {
             if (this.server && !this.server.isClosed()) {
                 this.server.close();
             }
 
-            var spec = this;
-
             var req = indexedDB.deleteDatabase(this.dbName);
-
-            req.onsuccess = function () {
-                done();
-            };
-
-            req.onerror = function () {
-                console.log('failed to delete db in afterEach', arguments, spec);
-            };
-
-            req.onblocked = function () {
-                console.log('db blocked', arguments);
-            };
         });

```

Comment:
I'd think it should still work with `done()` in `onerror` or `onblocked`, but gotcha re: unique DB name...